EmpleoDotNet/Models/EntityBase.cs

```diff
@@ -10,11 +8,14 @@ namespace EmpleoDotNet.Models
     /// </summary>
     public abstract class EntityBase
     {
+        protected EntityBase()
+        {
+            Created = DateTime.Today;
```

Comment:
Use `DateTime.Now` or `DateTime.UtcNow` instead. Today will always have the date at midnight